plugins/UsersManager/vue/src/PagedUsersList/PagedUsersList.vue

```diff
@@ -388,7 +388,6 @@
             `<strong>${affectedUsersCount}</strong>`,
           ))"
       ></h2>
-      <p>{{ translate('UsersManager_ConfirmWithPassword') }}</p>
```

Comment:
Can't comment on the lines below as they are not part of the change, but should the next `PasswordConfirmation` component with `v-model="showPasswordConfirmationForAnonymousAccess"` after this have `h2` headings instead of `h3`s? If we're touching things here we might as well adjust that?

Reply:
I guess the reason for using a h3 there, was that the text can be quite long and using h2, might break it into a lot of lines.

Reply:
Right, ok. The correct way would be to still use h2 tag with a class modifier to make the text smaller/look like h3 but keep the semantics. All good, out of scope here.